packages/argo-checkout/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -1,88 +1,82 @@
 import type {StandardApi} from '../standard';
 import type {ValueOrPromise} from '../shared';
 
-/** input given to the ShouldRender extension point (Checkout::PostPurchase::ShouldRender) */
+/** Input given to the ShouldRender extension point (Checkout::PostPurchase::ShouldRender) */
 export interface PostPurchaseShouldRenderApi
   extends StandardApi<'Checkout::PostPurchase::ShouldRender'> {
-  /** current cart being checked out */
-  checkout: Checkout;
+  /** Current cart being checked out */
+  initialPurchase: Purchase;
 
-  /** general purpose storage for extensions */
+  /** General purpose storage for extensions */
   storage: Storage;
 
-  /** shop where the checkout/order is from */
+  /** Shop where the checkout/order is from */
   shop: Shop;
 }
 
-/** output expected from the ShouldRender extension point (Checkout::PostPurchase::ShouldRender) */
+/** Output expected from the ShouldRender extension point (Checkout::PostPurchase::ShouldRender) */
 export type PostPurchaseShouldRenderResult = ValueOrPromise<{
-  /** whether or not to render the post-purchase page */
+  /** Whether or not to render the post-purchase page */
   render: boolean;
 }>;
 
-/** input given to the render extension point (Checkout::PostPurchase::Render) */
+/** Input given to the render extension point (Checkout::PostPurchase::Render) */
 export interface PostPurchaseRenderApi
   extends StandardApi<'Checkout::PostPurchase::Render'> {
-  /** initial purchase */
-  order: Order;
-
-  /** general purpose storage for extensions */
+  /** Initial purchase */
+  order: Purchase;
```

Comment:
it's not `order` here, it's also `initialPurchase`

Reply:
Good catch! :blobheart: